mdlight/server.py

```diff
@@ -27,6 +28,21 @@
 
 
 import mdlight.index.tree
+from mdlight.index.pages import MarkdownPage, GraphvizPage
+
+
+_REQUIRED_BINARIES = [
+    MarkdownPage.BINARY_NAME,
+    GraphvizPage.BINARY_NAME,
+]
+
+
+def __check_binary(binary):
+    for path in os.environ["PATH"].split(os.pathsep):
```

Comment:
There is a python3, let use `shutil.which` instead. That would be more understandable and portable for sure.